src/js/views/settingsView.js

```diff
@@ -0,0 +1,124 @@
+var Marionette = require('backbone.marionette'),
+    _ = require('underscore'),
+    $ = require('jquery'),
+    bindAll = require('lodash.bindall');
```

Comment:
I might just be being nit picky here but the style that I have seen so far is to have a require on individual lines with `var`s. If we want to keep this then perhaps template should also be added here?

    bindAll = require('lodash.bindall'),
    template = require('../../templates/settings.tpl');